### PR TITLE
BC-7030 - new sidebar

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Description

To ensure a consistent layout and create the possibility for broader content width, we want to refactor the legacy sidebar to match the new vue design.

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-7030
https://github.com/hpi-schul-cloud/nuxt-client/pull/3193

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: static/styles/lib/loggedin.scss [hunk 1]
```diff
@@ -390,11 +391,17 @@ header {
...
+	transition: 0.2s cubic-bezier(0.4, 0, 0.2, 1);
 
-	@include media-breakpoint-down(md) {
-		padding-left: 60px;
+	// new sidebar
+	@media only screen and (max-width: 1279px) {
+		padding: 0;
 	}
 
+	// @include media-breakpoint-down(md) {
```
Comment (MartinSchuhmacher) [inline static/styles/lib/loggedin.scss:401]: remove comment?

File: static/styles/lib/sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+				}
+			}
+
+			.sidebar-item {
+				display: flex;
+				align-items: center;
+				width: 100%;
+				height: 48px;
+				color: $secondaryColor;
+				// fill: $secondaryColor;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/sidebar.scss:143]: remove comment?

File: views/lib/loggedin.hbs [hunk 3]
```diff
@@ -45,7 +45,8 @@
             <input type="hidden" id="schuljahrtransfer" value="{{currentRole}}"/>
         {{/if}}
 
-        <div class="navtooltips"></div>
+        {{#embed "lib/sidebar"}}{{/embed}}
+        {{!-- <div class="navtooltips"></div>
```
Comment (MartinSchuhmacher) [inline views/lib/loggedin.hbs:49]: why leaving all the comments? also area above

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]
System > GitHub opens in same tab, should be new tab as the others

File: static/scripts/loggedin.js [hunk 4]
```diff
@@ -76,6 +159,35 @@ $(document).ready(() => {
 });
 
 $(document).ready(function () {
+    // new sidebar
+    var groupToggleBtns = document.querySelectorAll('.group-toggle-btn');
```
[the remark this answers is not in the document]
Comment (odalys-dataport) [reply static/scripts/loggedin.js:163]: the side product of copying lines :D

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**